recipe/meta.yaml

```diff
@@ -14,18 +14,21 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
+  # noarch: python
   number: 0
-  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
-  skip: true  # [osx and py35]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: True  # [py2k]
 
 requirements:
-  build:
+  host:
     - python
     - setuptools
+    - setuptools_scm
+    - pip
   run:
     - python
     - astropy >=1.0.2
-    - emcee >=2.2.0
+    - emcee >=2.2.0,<3.0
```

Comment:
@zblz - Do you need this `emcee<3.0` constraint? Or can you remove it? I think they tried to be backward compatible mostly?

If you need this, we would have to revert back to the older emcee in the Gammapy env we advertise to users here:

https://github.com/gammapy/gammapy-webpage/blob/9b39513f596a5b7b56fe5a2b6414b21ddde79c41/download/install/gammapy-0.15-environment.yml#L34

For now, we keep the `pip install naima`, but we'd like to change to `conda install naima` to have a faster and more consistent install if possible.

cc @adonath